mobile-inbound-api/v1/openapi.yaml

```diff
@@ -0,0 +1,114 @@
+openapi: 3.0.0
+info:
+  title: Mobile Inbound SMS API
+  description: |
+    The Mobile Inbound SMS API enables you to receive messages from your
+    customers sent to phone numbers you have registered with Tyntec.
+
+    For this service you will need to provide tyntec with the URL
+    (`callbackUrl`) of your webserver, e.g.
+    `https://rest.customer.com/inboundmessages/`.
+
+    tyntec's application will do either a `POST` or `GET` call to you
+    server; this can be configured by tyntec.
+
+    tyntec's application will retry delivery every 10 seconds in case your
+    application does not accept the inbound message. Retries are paused for
+    10 minutes after every 100 consecutive unsuccessful delivery attempts.
+    tyntec's application will retry for a maximum of 48 hours.
+
+  contact:
+    name: tyntec API Support
+    url: http://www.tyntec.com/support
+    email: support@tyntec.com
+  version: '1.0'
+  x-repository: https://github.com/tyntec/api-collection/blob/master/mobile-inboud-api
+  x-major-version: v1
+servers:
+  - url: https://{defaultHost}
+    variables:
+      defaultHost:
+        default: www.example.com
+paths:
+components:
+  schemas:
```

Comment:
In general I miss examples eg:

        from:
          type: string
          example: 49176244234